lib/activerecord/all/activerecord.rbi

```diff
@@ -128,7 +128,7 @@ module ActiveRecord::Associations::ClassMethods
       foreign_type: T.nilable(T.any(Symbol, String)),
       inverse_of: T.nilable(T.any(Symbol, String)),
       optional: T.nilable(T::Boolean),
-      polymorphic: T.nilable(T.any(Symbol, String)),
+      polymorphic: T.nilable(T.any(Symbol, String, T::Boolean)),
```

Comment:
Thanks for catching this! 

I actually think it only needs to allow Boolean values; Strings or Symbols don't really make sense here, AFAIK.

```suggestion
      polymorphic: T.nilable(T::Boolean),
```

Reply:
You are right.
I fixed it !